src/BrowserKitDriver.php

```diff
@@ -413,7 +413,24 @@ public function getValue($xpath)
      */
     public function setValue($xpath, $value)
     {
-        $this->getFormField($xpath)->setValue($value);
+        $formField = $this->getFormField($xpath);
+
+        try {
```

Comment:
I know, but as you can see in `getSelectValuesByOptionTest` method, that you've added you're collecting `value` attributes for options, which title matched. If you have:

```html
<select name="test">
  <option>title one</option>
  <option>title two</option>
</select>
```

that method won't help much.